test/mpi/runtests.in

```diff
@@ -437,7 +436,7 @@ sub RunList {
             # just skip, do not complain missing unless it is "testlist"
             next;
         }
-        # FIXME: keeping the indentation to facilitate review, indent back on merge
```

Comment:
removing this comment should be in the previous commit?

Reply:
Sure. Let me delete the first commit as well. I can always rebase.